db/migrate/20160523155343_add_enabled_column_to_service.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnabledColumnToService < ActiveRecord::Migration
+  def change
+    add_column :services, :enabled, :boolean, default: false, null: false
```

Comment:
So I tried adding an index to the 'enabled' column, but running `EXPLAIN ANALYZE` in dev seems to indicate that the query plan does not make use of the index. Presumably because over 25% of all services are enabled, so too many rows are returned and a sequential scan proves to be more performant.

I also tried changing the existing indexes for the Service table to be partial indexes, but did not see any improvement in query times. I'm aware that the query planner might behave differently in the production environment, but I'm happy to just merge this PR as is (ie not adding any extra index), and revisit this sort of optimisation in future when required.